### PR TITLE
feat(billingbudgets): initial generation of billing budget

### PR DESCRIPTION
**Pull request by busunkim96**

### CODE REVIEW

File: billingbudgets/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+Python Client for Cloud Billing Budget API (`Alpha`_)
+=====================================================
```
Comment (tseaver) [inline billingbudgets/README.rst]: Needs a top-level summary description.

File: billingbudgets/docs/README.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Python Client for Cloud Billing Budget API (`Alpha`_)
```
Comment (tseaver) [inline billingbudgets/docs/README.rst]: This file should be a symlink to the one in the directory above.

File: billingbudgets/.repo-metadata.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "billingbudgets",
+  "name_pretty": "Cloud Billing Budget",
+  "product_documentation": "https://cloud.google.com/billing/docs/apis",
```
Comment (tseaver) [inline billingbudgets/.repo-metadata.json]: The budget-oriented APIs we are wrapping here aren't documented below this URL.

File: billingbudgets/README.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+`Cloud Billing Budget API`_:
+
+- `Client Library Documentation`_
+- `Product Documentation`_
+
+.. _Alpha: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
+.. _Cloud Billing Budget API: https://cloud.google.com/billingbudgets
```
Comment (tseaver) [inline billingbudgets/README.rst]: 404

File: billingbudgets/README.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+.. _Client Library Documentation: https://googleapis.dev/python/billingbudgets/latest
+.. _Product Documentation:  https://cloud.google.com/billingbudgets
```
Comment (tseaver) [inline billingbudgets/README.rst]: 404

File: billingbudgets/README.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+~~~~~~~~~~
+
+-  Read the `Client Library Documentation`_ for Cloud Billing Budget API
+   API to see other available methods on the client.
+-  Read the `Cloud Billing Budget API Product documentation`_ to learn
+   more about the product and see How-to Guides.
+-  View this `repository’s main README`_ to see the full list of Cloud
+   APIs that we cover.
+
+.. _Cloud Billing Budget API Product documentation:  https://cloud.google.com/billingbudgets
```
Comment (tseaver) [inline billingbudgets/README.rst]: 404

File: billingbudgets/setup.py [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    author_email="googleapis-packages@google.com",
+    license="Apache 2.0",
+    url="https://github.com/googleapis/google-cloud-python",
+    classifiers=[
+        release_status,
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
```
Comment (tseaver) [inline billingbudgets/setup.py]: Should we be promising Python2 support for newly-generated client libraries?

File: billingbudgets/.repo-metadata.json (hunk 3 above)
Comment (busunkim96) [reply billingbudgets/.repo-metadata.json]: I think the actual docs may not be published. Checking with the tech writer to see where the docs will eventually be published

File: billingbudgets/README.rst (hunk 1 above)
Comment (busunkim96) [reply billingbudgets/README.rst]: [`billingbudgets.yaml`](https://github.com/googleapis/googleapis/blob/3fb0873b5f8a4cf5be4f848d20e5ccb2bdee1a67/google/cloud/billing/budgets/billingbudgets.yaml) is missing a summary line. I've asked the API team to add one.

File: billingbudgets/docs/README.rst (hunk 2 above)
Comment (busunkim96) [reply billingbudgets/docs/README.rst]: Done

File: billingbudgets/setup.py (hunk 7 above)
Comment (busunkim96) [reply billingbudgets/setup.py]: Dropped 2 and 2.7 classifiers

File: billingbudgets/.repo-metadata.json (hunk 3 above)
Comment (busunkim96) [reply billingbudgets/.repo-metadata.json]: Confirmed that the link will be https://cloud.google.com/billing/docs/how-to/budget-api-overview

File: billingbudgets/README.rst (hunk 1 above)
Comment (tseaver) [reply billingbudgets/README.rst]: Do you want to wait on that change and update the PR with the same text?

Comment (busunkim96) [reply billingbudgets/README.rst]: Yes I'll keep this open until I see the change to the protos

**Pull request merged**